.gitignore

```diff
@@ -270,4 +270,4 @@ paket-files/
 
 # Environment Specific app settings
 appsettings.*.json
-secrets.config
+secrets.config
```

Comment:
This file should not exist in the repo at all. Could you remove it here and delete the file if you have it on your end?

Reply:
Oh I'm also wondering why this thing is here. I copied and pasted the original .gitignore from develop branch to the file on my end, but it still says the `secrets.config` is changed. How can I remove it here and delete the file on my end as you said?

Reply:
I don't know what you mean "copied from the develop branch"

Just use `git checkout origin/develop -- .gitignore` to overwrite the file from develop.

Reply:
Oh Awesome! I did it. Thanks!